r10k.gemspec

```
@@ -24,21 +24,15 @@ Gem::Specification.new do |s|
 
   s.add_dependency 'colored2', '~> 4.0'
   s.add_dependency 'cri', '>= 2.15.10'
-
   s.add_dependency 'log4r',     '1.1.10'
   s.add_dependency 'multi_json', '~> 1.10'
-
-  s.add_dependency 'puppet_forge', '~> 6.0'
-
+  s.add_dependency 'puppet_forge', '>= 4.1.0'
   s.add_dependency 'gettext-setup', '>=0.24', '<2.0'
-
   s.add_dependency 'jwt', '>= 2.2.3', '< 3'
-  s.add_dependency 'minitar', '~> 1.0', '>= 1.0.2'
+  s.add_dependency 'minitar', '>= 0.9'
```

Did you test if this works with minitar 0.x and 1.x? I thought that requires some code changes. And I think the versioning should be a bit more strict, to ensure that we don't pull in major releases without testing them first:
```suggestion
  s.add_dependency 'minitar', '>= 0.9', '< 2'
```

Then dependabot will provide a PR when there's a new version and CI will automatically test it.

---

I ran the r10k tests.

FWIW, this is what I was referencing in the commit message: https://github.com/halostatue/minitar/blob/v0.9/lib/minitar.rb#L11